files/icons/awesome.toml

```diff
@@ -47,10 +47,10 @@ memory_mem = "\uf2db" # fa-microchip
 memory_swap = "\uf0a0" # fa-hdd-o
 mouse = "\uf245" # fa-mouse-pointer
 music = "\uf001" # fa-music
-music_next = "\uf061" # fa-arrow-right
+music_next = "\uf050" # fa-forward-fast
 music_pause = "\uf04c" # fa-pause
 music_play = "\uf04b" # fa-play
-music_prev = "\uf060" # fa-arrow-left
+music_prev = "\uf049" # fa-backward-fast
```

Comment:
I didn't mean to add these font changes in this commit, but now that I have... Any thoughts on swapping out the plain arrows for the media player style forward and back buttons?

I know I can just override them in my config, if I don't like the defaults, but I thought that these are more appropriate.

Reply:
Yeah, `fa-backward-step` / `fa-forward-step` might me the most appropriate. Can you open a new PR for that?